config/desktop/jammy/appgroups/editors/packages

```diff
@@ -2,5 +2,4 @@ emacs
 geany
 gedit
 gedit-plugins
-vim
```

Comment:
any reason for taking this out? at least replace it with `vim-tiny` if its a size thing (I use Vim)